<?xml version='1.0' encoding='UTF-8' ?>
<testsuite name="com.otop.SimonDice.TestLista" tests="1" failures="1" errors="0" skipped="0" time="4.828" timestamp="2023-04-21T16:46:33" hostname="localhost">
  <properties>
    <property name="device" value="Pixel_6_API_30(AVD) - 11" />
    <property name="flavor" value="" />
    <property name="project" value=":app" />
  </properties>
  <testcase name="when_top_is_update" classname="com.otop.SimonDice.TestLista" time="4.183">
    <failure>java.lang.NullPointerException: Cannot run onActivity since Activity has been destroyed already
at androidx.test.internal.util.Checks.checkNotNull(Checks.java:45)
at androidx.test.core.app.ActivityScenario.lambda$onActivity$2$androidx-test-core-app-ActivityScenario(ActivityScenario.java:787)
at androidx.test.core.app.ActivityScenario$$ExternalSyntheticLambda2.run(Unknown Source:4)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:462)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at android.app.Instrumentation$SyncRunnable.run(Instrumentation.java:2223)
at android.os.Handler.handleCallback(Handler.java:938)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:223)
at android.app.ActivityThread.main(ActivityThread.java:7656)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)</failure>
  </testcase>
</testsuite>